runner/runners/invoke.go

```diff
@@ -193,10 +199,8 @@ func (inv *Invoker) run(cmd *runner.Command, id runner.RunID, abortCh chan struc
 				log.Errorf("Error canceling checkout: %s", err)
 			}
 			if err := <-checkoutCh; err != nil {
-				// If there was an error there should be no lingering gitdb locks, so return.
-				return
+				log.Errorf("Checkout errored: %s", err)
```

Comment:
Don't have full context on the deleted comment and return - why is the return safe to get rid of?

Reply:
The only difference is we now will still release the checkout (i.e. unlock the DB.workTreeLock) even if there was an error checking it out. I don't know why we assumed there wouldn't be any lingering gitdb locks if the checkout failed in the first place.